dev-docs/monitoring-suggestions.md

```diff
@@ -0,0 +1,46 @@
+# Monitoring Suggestions
+
+Taskcluster has several background processes that you should ensure are running on a schedule. Any of the following will generate messages
+of the form:
+
+```json
+{
+  "Type": "monitor.periodic",
+  "Logger": "<Logger>",
+  "Fields": {
+    "name": "<Name>"
+  }
+}
+```
+
+They will also have Fields for `status`, `duration`, and a serialized `error` if an error occured.
+
+The processes that have `continuous` for their dedaline and schedule run every few minutes and should complete fairly quickly. The rest
+have their schedules and maximum allowed duration defined here.
```

Comment:
It's probably worth including a markdown comment indicating that this file is automatically generated and should not be modified.